src/web_interface/rest/rest_missing_analyses.py

```diff
@@ -9,14 +9,24 @@
 from web_interface.security.privileges import PRIVILEGES
 
 
+api = Namespace('rest/missing', description='Search the database for missing entries')
+
+
+@api.route('', doc={'description': ''})
 class RestMissingAnalyses(Resource):
     URL = '/rest/missing'
 
-    def __init__(self, **kwargs):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
         self.config = kwargs.get('config', None)
 
     @roles_accepted(*PRIVILEGES['delete'])
+    @api.doc(responses={200: 'Success', 400: 'Unknown'})
     def get(self):
+        '''
+        Search for missing entries
+        Search for missing files or missing analyses
```

Comment:
This endpoint has some additional functionality that is still missing from the REST docs
```suggestion
        Search for missing or orphaned files and missing or failed analyses
```